Sources/Routing/Register/PathComponent.swift

```diff
@@ -28,6 +31,7 @@ extension Array where Element == PathComponent {
             case .constant(let s): return s
             case .parameter(let p): return ":\(p)"
             case .anything: return "*"
+            case .catchall: return "*"
```

Comment:
While I understand the reason for having an asterisk here, I think it might cause some confusion sooner or later. I am not entirely sure what we would use instead though. Maybe `sudo` 😜. To be honest, something like `*->` might work.